src/panels/config/devices/ha-config-device-page.ts

```diff
@@ -148,21 +158,58 @@ export class HaConfigDevicePage extends LitElement {
         ></paper-icon-button>
 
         <div class="container">
-          <div class="left">
-            <div class="device-info">
-              ${
-                this.narrow
-                  ? ""
-                  : html`
+          <div class="header fullwidth">              
+            ${
+              this.narrow
+                ? ""
+                : html`
+                    <div>
                       <h1>${computeDeviceName(device, this.hass)}</h1>
-                    `
-              }
-              <ha-device-card
+                      ${areaName
+                        ? this.hass.localize(
+                            "ui.panel.config.integrations.config_entry.area",
+                            "area",
+                            areaName
+                          )
+                        : ""}
+                    </div>
+                  `
+            }
+                <div class="header-right">
+                  ${
+                    batteryState
+                      ? html`
+                          <div class="battery">
+                            ${batteryState.state}%
+                            <ha-state-icon
+                              .hass=${this.hass!}
+                              .stateObj=${batteryState}
+                            ></ha-state-icon>
+                          </div>
+                        `
+                      : ""
+                  }
+                  <img
+                    src="https://deploy-preview-61--home-assistant-brands.netlify.com/${
```

Comment:
Needs update.

Reply:
The upstream PRs / changes have been deployed. So it can nog be updated to the production URL.